pkg/maglev/maglev_test.go

```diff
@@ -30,26 +30,32 @@ type MaglevTestSuite struct{}
 var _ = Suite(&MaglevTestSuite{})
 
 func (s *MaglevTestSuite) SetUpTest(c *C) {
-	InitMaglevSeeds(DefaultHashSeed)
+	if err := InitMaglevSeeds(DefaultHashSeed); err != nil {
+		panic(err)
```

Comment:
Nit: use `c.Fatal` here, I don't think the stack trace is relevant here

```suggestion
		c.Fatal(err)
```